cores/arduino/api

```diff
@@ -1 +1 @@
-../../../ArduinoCore-API/api/
\ No newline at end of file
+/home/pennam/Arduino/API/api/
```

Comment:
```suggestion
../../../ArduinoCore-API/api/
```

Revert incorrect change to symlink target path.

Reply:
done! Thanks @per1234